release/Makefile.in

```diff
@@ -28,5 +29,6 @@ installmoloch:
 	@mkdir -p $(etcdir) @prefix@/logs
 	sed -e "s,BUILD_ARKIME_INSTALL_DIR,@prefix@,g" < arkime_update_geo.sh > @prefix@/bin/moloch_update_geo.sh
 	sed -e "s,BUILD_ARKIME_INSTALL_DIR,@prefix@,g" < arkime_add_user.sh > @prefix@/bin/moloch_add_user.sh
+	sed -e "s,BUILD_ARKIME_INSTALL_DIR,@prefix@,g" < arkime_config_interfaces.sh > @prefix@/bin/arkime_config_interfaces.sh
```

Comment:
These 2 in the moloch section aren't needed since Configure for moloch just tells you not to run

Reply:
Are you sure? Because without these additions, the new file "arkime_config_interfaces.sh" is not copied into the ARKIME_INSTALL_DIR/bin directory.

Reply:
yep, the moloch install doesn't use anything called arkime* and actually doesn't install anything, the Configure script is empty